common/protos/proto_utils.ts

```diff
@@ -0,0 +1,29 @@
+import { util } from "protobufjs";
+
+export class ProtoUtils {
+  public static decode64(value: string): Uint8Array {
+    const buf = new Uint8Array(util.base64.length(value));
+    util.base64.decode(value, buf, 0);
+    return buf;
+  }
+
+  public static encode64(value: Uint8Array): string {
+    return util.base64.encode(value, 0, value.length);
+  }
+
+  public static decode<Proto>(protoType: new () => Proto, encodedValue: string): Proto {
+    if (!encodedValue || encodedValue === "" || encodedValue === "_") {
```

Comment:
this stuff seems bug prone, do we need it?

Reply:
Removed, no longer needed